executor/show_stats_test.go

```diff
@@ -62,3 +63,23 @@ func (s *testSuite) TestShowStatsBuckets(c *C) {
 	result = tk.MustQuery("show stats_buckets where column_name = 'idx'")
 	result.Check(testkit.Rows("test t idx 1 0 1 1 (1, 1) (1, 1)"))
 }
+
+func (s *testSuite) TestShowStatsHealthy(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+	tk.MustExec("drop table if exists t")
+	tk.MustExec("create table t (a int)")
+	tk.MustExec("create index idx on t(a)")
+	tk.MustExec("analyze table t")
+	tk.MustExec("insert into t values (1), (2)")
+	do, _ := tidb.GetDomain(s.store)
+	do.StatsHandle().DumpStatsDeltaToKV()
+	tk.MustExec("analyze table t")
+	tk.MustQuery("show stats_healthy").Check(testkit.Rows("test t 100"))
+	tk.MustExec("insert into t values (3), (4), (5), (6), (7), (8), (9), (10)")
+	do.StatsHandle().DumpStatsDeltaToKV()
+	do.StatsHandle().Update(do.InfoSchema())
+	tk.MustQuery("show stats_healthy").Check(testkit.Rows("test t 19"))
+	tk.MustExec("analyze table t")
+	tk.MustQuery("show stats_healthy").Check(testkit.Rows("test t 100"))
```

Comment:
add a `%`?

Reply:
But it is quite weird to have something like `-10%`?